fmriprep/workflows/bold/base.py

```diff
@@ -259,6 +260,26 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
         run_stc = True
         multiecho = False
     else:
+        # Find associated sbref, if possible
+        entities = layout.parse_file_entities(ref_file)
+        entities['type'] = 'sbref'
+        files = layout.get(**entities)
+        refbase = os.path.basename(ref_file)
+        if files and multiecho:
```

Comment:
Just to double check: is the conflict for all ME or only if ``--t2-coreg`` is set?

Reply:
The conflict is basically that things are going to change in #1263, and the multi-echo w/ sbref case doesn't have a full set of sbrefs, so let's not mess with it now.